samples/kaspresso-sample/build.gradle.kts

```diff
@@ -9,8 +9,37 @@ android {
         testInstrumentationRunnerArguments["clearPackageData"] = "true"
     }
 
+    sourceSets {
+        val main by getting {
+            java.srcDir("src/main/kotlin")
+        }
+
+        //configure shared test folder
+        val sharedTestFolder = "src/sharedTest/kotlin"
```

Comment:
Not sure whether this is enough to make the shared tests automatically run in the CI not only as Instrumentation tests but also as JVM Unit Tests. Otherwise, we need to add the following command to the job:

 on the JVM you need to run:
 ```
 ./gradlew :samples:kaspresso-sample:testDebugUnitTest --tests "com.kaspersky.kaspressample.sharedtest.SharedTest"
 ```


 For instance, to run those tests device/emulator, you need to run:
 ```
 ./gradlew :samples:kaspresso-sample:connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.package=com.kaspersky.kaspressample.sharedtest
 ```

Reply:
@matzuk @RuslanMingaliev @eakurnikov 
Please consider to add these tasks to the CI job whenever this gets merged :)